troposphere/static/js/components/common/InstanceAccessListView.jsx

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import Backbone from "backbone";
+import subscribe from 'utilities/subscribe';
+import Glyphicon from "components/common/Glyphicon";
+import context from "context";
+import featureFlags from "utilities/featureFlags";
+
+import actions from "actions";
+/**
+ * WIP:
+ * - User is 'selected' in the chosen, but the query is not updated
+ */
+const InstanceAccessListView = React.createClass({
+    displayName: "InstanceAccessListView",
+
+
+    propTypes: {
+        instance: React.PropTypes.instanceOf(Backbone.Model),
+    },
+    getInitialState: function() {
+        return {
+        }
+    },
+    //
+    // Render
+    // ------
+    //
+    renderRequest: function() {
+        let { UserStore } = this.props.subscriptions,
+            options = UserStore.getAllPages(),
+            instance_ip = this.props.instance.get('ip_address'),
+            current = UserStore.getUsername(this.state.username);
```

Comment:
ToDo: (@cdosborn ) mentioned that this could be done differently by:
1) Don't rely on `UserStore.getAll()` and instead expect user to "fill out the dropdown" and use API lookups (and `queryModels`) to find first 1000 of users.
2) Lookup the individual user, rather than expect to find them within the UserStore.